cmd/wallet-js-sdk/src/did/didmanager.js

```diff
@@ -92,10 +100,24 @@ export class DIDManager {
           keyType: DEFAULT_KEY_TYPE,
           update: true,
         },
+        {
+          id: keyAgreementKeySet.keyID,
+          type: keyAgreementType,
+          value: keyAgreementKeySet.publicKey,
+          encoding: "JWK",
+          keyType: keyAgreementKeyType,
+          purposes: ["keyAgreement"],
+        },
       ],
     };
 
-    let content = await this.agent.didclient.createOrbDID(createDIDRequest);
+    let content
+    try {
+      content = await this.agent.didclient.createOrbDID(createDIDRequest);
+    } catch (e) {
+      console.error('failed to create orbDID',e)
```

Comment:
you are catching error and doing nothing, this will cause panic in browser (JS error)

Reply:
removed try/catch bloc